pkgs/applications/networking/irc/kvirc/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub
```

Comment:
Did you want to add optional `perl`, `python`, and `kde` support?

Reply:
No.